definitions/private-law/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -263,5 +263,25 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "LiveFrom": "08/05/2024",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "citizenPcqUpdate",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "citizen",
+          "caseworker-privatelaw-systemupdate"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "caseworker-privatelaw-readonly"
+        ],
+        "CRUD": "R"
+      }
+    ]
   }
```

Comment:
Move this to a new folder and a file please. this file is overloaded.
Create a folder called pcq and move it over there please.